iroh-p2p/src/node.rs

```diff
@@ -1445,6 +1445,7 @@ mod tests {
         Ok(())
     }
 
+    #[ignore = "flakey"]
```

Comment:
is there an issue to track this, why is there no todo to review this test and fix it?

Reply:
Sure thing: https://github.com/n0-computer/iroh/issues/518